netmaster/mastercfg/policyState.go

```diff
@@ -293,6 +293,13 @@ func (gp *EpgPolicy) createOfnetRule(rule *contivModel.Rule, dir string) (*ofnet
 		return nil, err
 	}
 
+	// Send AddRule to netplugin agents
+	err = addPolicyRuleState(ofnetRule)
```

Comment:
When return values are not used outside of the `if` statement immediately following a function call, it's convention to write it like this to avoid putting the variables into the function scope at all:

```go
if err := some_func(); err != nil {
   // err exists here but not in the scope of the function this code is in
}
```

This also means you don't have to worry about using `:=` or `=` in the assignment.  You can always use `:=`

`processStateEvent()` in `state_event.go` has some examples of doing it with multiple return values